### PR TITLE
Typos and fixes

### PR DESCRIPTION
**Pull request by bentley**
First full read in a while.

Chapter 26 quotes the body of a letter, but without salutation or anything else. I don’t think letter semantics make sense there, so I removed them. (If this *does* need letter semantics, then does also chapter 29’s paragraph starting “I cannot understand what is the matter with papa,” structured exactly the same way?)

All other non‐editorial commits bring us closer to the Hathitrust scans.

### CODE REVIEW

File: src/epub/text/chapter-26.xhtml [hunk 1]
```diff
@@ -12,9 +12,7 @@
...
-			</blockquote>
+			<p>“I know,” wrote Brian, “that now you have taken up the affair, you will not stop until you find out everything, so, as I want the matter to rest as at present, I will anticipate you, and reveal all. You were right in your conjecture that I knew something likely to lead to the detection of Whyte’s murderer; but when I tell you my reasons for keeping such a thing secret, I am sure you will not blame me. Mind you, I do not say that I know who committed the murder; but I have suspicions⁠—very strong suspicions⁠—and I wish to God Rosanna Moore had died before she told me what she did. However, I will tell you all, and leave you to judge as to whether I was justified in concealing what I was told. I will call at your office some time next week, and then you will learn everything that Rosanna Moore told me; but once that you are possessed of the knowledge you will pity me.”</p>
```
Comment (acabal) [inline src/epub/text/chapter-26.xhtml]: In the page scans this is actually part of the preceding paragraph. Can you join them?

**Review by acabal — CHANGES_REQUESTED** [review]
Great fixes! There's one small note here, can you take a look?

File: src/epub/text/chapter-26.xhtml (hunk 1 above)
Comment (bentley) [reply src/epub/text/chapter-26.xhtml]: Done in dbec3a53ea64fcf13c87fe088666e12d5373d2ea.

**Comment by acabal** [comment]
Great, thanks!

**Pull request merged**